theme/src/components/layout.js

```diff
@@ -28,30 +34,75 @@ function Layout({children, pageContext}) {
         <Box display={['none', null, null, 'block']}>
           <Sidebar />
         </Box>
-        <Container>
-          <H1>{title}</H1>
-
-          {status || source ? (
-            <Flex mb={4} alignItems="center">
-              {status ? <StatusLabel status={status} /> : null}
-              <Box mx="auto" />
-              {source ? <SourceLink href={source} /> : null}
-            </Flex>
-          ) : null}
-
+        <Grid
+          id="skip-nav"
+          maxWidth="100%"
+          gridTemplateColumns={['100%', null, 'minmax(0, 65ch) 220px']}
+          gridTemplateAreas={[
+            '"heading" "content"',
+            null,
+            '"heading ." "content table-of-contents"',
+          ]}
+          gridColumnGap={[null, null, 6, 7]}
+          gridRowGap={3}
+          mx="auto"
+          p={[5, 6, null, 7]}
+          css={{alignItems: 'start', alignSelf: 'start'}}
+        >
+          <BorderBox
+            css={{gridArea: 'heading'}}
+            border={0}
+            borderBottom={1}
+            borderRadius={0}
+            pb={2}
+          >
+            <Heading>{title}</Heading>
+          </BorderBox>
           {pageContext.tableOfContents.items ? (
-            <TableOfContents items={pageContext.tableOfContents.items} />
+            <Position
+              display={['none', null, 'block']}
+              css={{gridArea: 'table-of-contents', overflow: 'auto'}}
+              position="sticky"
+              top={HEADER_HEIGHT + 24}
+              maxHeight={`calc(100vh - ${HEADER_HEIGHT}px - 24px)`}
+            >
+              <Text display="inline-block" fontWeight="bold" mb={1}>
+                Contents
```

Comment:
What do you think about calling this `Table of Contents` instead?
```suggestion
                Table of Contents
```

Reply:
That's fine with me. I just copied what @yaili had in her Figma document. I'll let her weigh in on this :)

Reply:
Also, to be super nit picky, if we go with "table of contents," should use sentence case ("Table of contents") since that's how we usually do headings?

Reply:
Yes, Table of contents :)